### PR TITLE
ADAPT-151 Fix the create pages function that came out of the box with gatsby (clean)

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# Summary
- Fix the create pages function that came out of box with the gatsby-blog-starter
- [ADAPT-151](https://stanfordits.atlassian.net/browse/ADAPT-151)

# Needed By (Date)
- ASAP?

# Urgency
- High — potential blocker for deploying to Netlify

# Steps to Test

1. Pull branch to local
2. `npm install` the node_modules
3. `gatsby develop`
3. View results at http://localhost:8000

# Affected Projects or Products
- [ADAPT-162](https://stanfordits.atlassian.net/browse/ADAPT-162)

# Associated Issues and/or People
- @buttonwillowsix 
- @katrialesser 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v8.16.0
```
Comment (sherakama) [inline .nvmrc]: Node is on 13 now. This should probably be up to 11 at least. 12-13 add some new things that might cause breakage but 8 is pretty old.

Comment (sherakama) [reply .nvmrc]: Guess it isn't THAT old:

Node.js 8.16.0 | Carbon | 2019-04-16
-- | -- | --

https://nodejs.org/en/download/releases/

**Comment by sherakama** [comment]
Don't forget to rebuild and run `npm install` after setting to v13.7.0 as it will change everything in your lock file.

File: .nvmrc (hunk 1 above)
Comment (rebeccahongsf) [reply .nvmrc]: Whoops. Great catch @sherakama ! Thank you :)

**Comment by sherakama** [comment]
I am guessing this can be closed now?

**Pull request closed without merging**